### PR TITLE
[zh-cn]: create the translation of `navigator.pdfViewerEnabled` property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/navigator/pdfviewerenabled/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  sourceCommit: b0c8b07682c8d2cecc544f60468f3cf6fc20ac99
+---
+
+{{APIRef("HTML DOM")}}
+
+{{domxref("Navigator")}} 接口的只读属性 **`pdfViewerEnabled`** 用于指示浏览器是否支持在导航到 PDF 文件时以内联方式显示它们。
+
+如果浏览器不支持内联显示，则 PDF 文件将被下载，并可能由外部应用程序处理。
+
+> **注意：** 此方法取代了多种传统的判断浏览器是否支持内联显示 PDF 文件的方法。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/navigator/pdfviewerenabled/index.md:14]:
```suggestion
> **备注：** 此方法取代了多种传统的判断浏览器是否支持内联显示 PDF 文件的方法。
```

`> **Note:**` 应翻译为 `> **备注：**`

File: files/zh-cn/web/api/navigator/pdfviewerenabled/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: Navigator：pdfViewerEnabled 属性
+slug: Web/API/Navigator/pdfViewerEnabled
+l10n:
+  sourceCommit: b0c8b07682c8d2cecc544f60468f3cf6fc20ac99
+---
+
+{{APIRef("HTML DOM")}}
+
+{{domxref("Navigator")}} 接口的只读属性 **`pdfViewerEnabled`** 用于指示浏览器是否支持在导航到 PDF 文件时以内联方式显示它们。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/navigator/pdfviewerenabled/index.md:10]:
```suggestion
{{domxref("Navigator")}} 接口的 **`pdfViewerEnabled`** 只读属性用于指示浏览器是否支持在导航到 PDF 文件时以内联方式显示它们。
```

File: files/zh-cn/web/api/navigator/pdfviewerenabled/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+如果浏览器不支持内联显示，则 PDF 文件将被下载，并可能由外部应用程序处理。
+
+> **注意：** 此方法取代了多种传统的判断浏览器是否支持内联显示 PDF 文件的方法。
+
+## 值
+
+如果浏览器在导航到 PDF 文件时可以使用内置查看器（或 PDF 查看器扩展程序）；则该属性值为 `true`，否则为 `false`。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/pdfviewerenabled/index.md:18]:
```suggestion
如果浏览器在导航到 PDF 文件时可以使用内置查看器（或 PDF 查看器扩展程序），则该属性值为 `true`；否则为 `false`。
```

Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/pdfviewerenabled/index.md:18]:
```suggestion
如果浏览器在导航到 PDF 文件时可以使用内置查看器（或 PDF 查看器扩展程序），则该属性值为 `true`；否则为 `false`。
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
Some suggestions for your reference.

**Review by JasonLamv-t — APPROVED** [review]
Thank you! @T34-active @skyclouds2001

**Pull request merged**